Resources/Locale/en-US/chat/ui/emote-menu.ftl

```diff
@@ -1,3 +1,4 @@
 emote-menu-category-general = General
 emote-menu-category-vocal = Vocal
 emote-menu-category-hands = Hands
+emote-menu-category-global-custom = Global custom
```

Comment:
I'm still really not sure what "global" is supposed to mean in this context. Why can't it just be "Custom"?

Reply:
Same as at the other place, I can change it temporarily if necessary.